client/my-sites/patterns/home/style.scss

```diff
@@ -0,0 +1,58 @@
+@import "@automattic/components/src/styles/typography";
+@import "@wordpress/base-styles/breakpoints.scss";
+
+.is-section-patterns .layout__content {
+	padding: 0;
```

Comment:
I'm on board with the fact that we need a `padding: 0` rule. My point was that, in my testing, it's being overriden by that CSS rule. This is what the page looks like for me locally:
![calypso localhost_3000_patterns(iPhone XR)](https://github.com/Automattic/wp-calypso/assets/1101677/a86a150b-b815-4c41-ac93-434870751059)

Reply:
Sorry, I didn't get your point :( 

So in my PR we have always `padding: 0`, and it's necessary first of all for header.
`My point was that, in my testing, it's being overriden by that CSS rule. This is what the page looks like for me locally` - hm, can we have a call to show me this behaviour? For me it looks w/o padding and we me it's expected according to the CSS.
You propose to use `@media screen and (max-width: 782px)` what means - we have default padding for desktop, so it would be broken UI, so we can't use this media-query, right?

Reply:
You're right. This doesn't happen when I refresh the page with a mobile viewport from the start. It only happens when I open the dev tools, activate responsive mode, and go from "Medium Laptop" to a mobile viewport size.

In short, it looks like you're right – this isn't an issue!